.github/workflows/check_registry.yaml

```diff
@@ -32,4 +32,4 @@ jobs:
         id: build
         run: |
           make build
-          bash test/check_registry/check_registry.sh
\ No newline at end of file
+          make check_registry
```

Comment:
@rm3l the nightly run checks the stacks for possible changes in order to catch any bug caused from registry (3rd party side). We have already the `git_test.go` which runs alizer against all [test_alizer_resources](https://github.com/devfile-resources/alizer-test-resources). I don't think we need to run this check for all the PRs.

For the failure I have already created an issue here: https://github.com/devfile/api/issues/1248

Reply:
Ok, makes sense then.
I expected this workflow to fail earlier on this PR because of the previous permission error. So it could have been merged without noticing this error.
Just curious, where is that `git_test.go` file located? I couldn't find it anywhere..
(Not urgent - you can reply later when you're back to work ^^)

Reply:
Oh :( my bad. It is `test/apis/git_recognizer_test.go`